contracts/Bond.sol

```diff
@@ -275,18 +284,14 @@ contract Bond is
         @dev emits Payment event
         @param amount the number of payment tokens to pay
     */
-    function pay(uint256 amount) external nonReentrant beforeMaturity {
+    function pay(uint256 amount) external nonReentrant {
```

Comment:
what is the reasoning behind removing the maturity check here?

I do not like the unclear behavior of bond holders and issuer.

Are the bond holders supposed to hold these indefinitely in hopes of some additional payment?

Reply:
> what is the reasoning behind removing the maturity check here?

Jordan shared that DAOs might miss the payment by about a week. If the collateral is worth less than the payment, then it would be unoptimal for bondholders to redeem for collateral if they could redeem for the payment amount. This enables the situation where the DAO could have more time to repay if they really needed to. 

> Are the bond holders supposed to hold these indefinitely in hopes of some additional payment?

If the DAO claims they will be repaying they can. Or they can redeem

> I do not like the unclear behavior of bond holders and issuer.

I aggree - but I think it makes sense in this situation. A defaulted bond being paid will be an edge case

Reply:
Lets talk about this more at lunch

Reply:
we are accepting the unclear states in favor of this "soft" grace period.